src/posix.mak

```diff
@@ -277,11 +274,11 @@ ROOT_SRC = $(addprefix $(ROOT)/, array.h ctfloat.h file.h filename.h \
 GLUE_SRC = \
 	toir.h irstate.h iasm.c \
 	toelfdebug.d libelf.d scanelf.d libmach.d scanmach.d \
-	tk.c eh.c gluestub.d objc_glue.c
```

Comment:
Because I forgot to remove it when eh.c was converted to eh.d. GLUE_SRC is just the manifest.